mod_upload/controllers.py

```diff
@@ -222,6 +242,36 @@ def process_id(upload_id):
                     g.db.rollback()
                 # Move file
                 if db_committed:
+                    with open("issues.md", "r") as f:
+                        data = f.read()
+                    split_arr = data.split('**')
+                    content = split_arr[3] + split_arr[4] + \
+                        split_arr[5] + split_arr[6]
+                    if form.report.data == 'y':
+                        necessary_details = ('**{head}** \n '
+                                             'What platform did you use? '
+                                             '{platform} \n '
+                                             'What where the used arguments? '
+                                             '{arg} \n').format(
+                            head=split_arr[7], platform=form.platform.data,
+                            arg=form.parameters.data)
+                        sample_link = ('{server_id}/sample/'
+                                       '{sample_id}').format(
+                            server_id=config.get('SERVER_NAME',
+                                                 ''), sample_id=sample.id)
+                        video_details = ('**{head}** \n [Sample Link]'
+                                         '({link}) \n {notes} \n').format(
+                            head=split_arr[9], link=sample_link,
+                            notes=form.notes.data)
+                        issue_title = ('[BUG] CCExtractor version '
```

Comment:
I don't like the version mentions in here. It should be in the issue content, in the title it's only cluttering.